lib/wibox/init.lua

```diff
@@ -31,6 +31,174 @@ function wibox:set_widget(widget)
     self._drawable:set_widget(widget)
 end
 
+-- Import some drawin documentation
+
+--- Border width.
+--
+-- **Signal:**
+--
+--  * *property::border_width*
+--
+-- @property border_width
+-- @param integer
+
+--- Border color.
+--
+-- **Signal:**
+--
+--  * *property::border_color*
+--
+-- @property border_color
+-- @tparam string|color|gradient|pattern A color
```

Comment:
Nope, only strings forming a hexadecimal color (without any alpha component). Welcome to X11! :-)